pkg/node/node.go

```diff
@@ -99,6 +102,54 @@ func maxPods(instanceType *string) int {
 	return enis * (ips - 1)
 }
 
+// The calculation here is based on information found in the GKE documentation
+// here: https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-architecture
+// I think that it should also apply to AWS
+func reservedCPU(instanceType *string) string {
+	cores := InstanceCores[*instanceType]
+	reserved := 0.0
+	for i := 0; i < cores; i++ {
+		if i < 1 {
+			reserved += 60.0
+		} else if i < 2 {
+			reserved += 10.0
+		} else if i < 4 {
+			reserved += 5.0
+		} else {
+			reserved += 2.5
+		}
+	}
+	if reserved == 0.0 {
+		reserved = 60.0
+	}
+	return fmt.Sprintf("%.0fm", reserved)
+}
+
+// The calculation here is based on information found in the GKE documentation
+// here: https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-architecture
+// I think that it should also apply to AWS
+func reservedMemory(instanceType *string) string {
+	memory := InstanceMemory[*instanceType]
+	reserved := 0.0
+	for i := 0; i < memory; i++ {
```

Comment:
I think these if-else statement would be easier to read on as switch cases, specially if we'd use explicit ranges instead of relying on the previous missed branch.